tools/c7n_azure/tests/templates/sqlserver.json

```diff
@@ -1,22 +1,45 @@
 {
-  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
-  "contentVersion": "1.0.0.0",
-  "parameters": {
-  },
-  "variables": {
-  },
-  "resources": [
-    {
-      "apiVersion": "2015-05-01-preview",
-      "type": "Microsoft.Sql/servers",
-      "location": "EastUS2",
-      "name": "cctestsqlserver12262018",
-      "properties": {
-        "administratorLogin": "custodian",
-        "administratorLoginPassword": "Cust0dianPassw0rd" ,
-        "version": "12.0"
+   "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+   "contentVersion": "1.0.0.0",
+   "parameters": {
+      "servers_cctestsqlserver12262018_name": {
+         "defaultValue": "cctestsqlserver12262018",
```

Comment:
If this value has to be unique, can you please use uniqueString() (see storage.json)

Reply:
It is a specific name referenced in `test_sqlserver.py` when performing assertions. I think the #'s are a date.

Reply:
I did need to change the date when updating the ARM template though, so I should probably update that name to match.

Reply:
The problem with hardcoded name is that the only person can deploy this template.
If you do this, you won't need to change the name each time you update tests.
```
"name": "[concat('cctestsqlserver', uniqueString(resourceGroup().id))]",
```

In that case, value filter in tests needs to be updated to search for prefix instead of full match.

```
            'filters': [
                {'type': 'value',
                 'key': 'name',
                 'op': 'glob',
                 'value_type': 'normalize',
                 'value': 'cctstorage*'}],
```